config/boards/rock960.conf

```diff
@@ -0,0 +1,9 @@
+# Rockchip RK3399 core 2GB RAM SoC GBE eMMC USB3 USB-C WiFi/BT
```

Comment:
File extension must be `.csc` if there is no named active maintainer for this board.